test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
I don't think so, this is part of the preparation for testing quests. What is supposed to happen is that three users are invited to the party, they all accept, then they're invited to the quest and 2 accept and 1 doesn't.

I suspect the problem is with the registerNewUser function, somehow it's giving back the original user instead of a new one.